src/DssSpell.t.sol

```diff
@@ -538,14 +480,26 @@ contract DssSpellTest is DssSpellTestBase {
 
     function testAAVEDirectBarChange() public {
         DirectDepositLike join = DirectDepositLike(addr.addr("MCD_JOIN_DIRECT_AAVEV2_DAI"));
-        assertEq(join.bar(), 3.5 * 10**27 / 100);
+        DSTokenAbstract adai = DSTokenAbstract(join.adai());
+
+        assertEq(join.bar(), 2.75 * 10**27 / 100);
 
         vote(address(spell));
-        scheduleWaitAndCast(address(spell));
+        DssSpell(spell).schedule();
+
+        // bar should now be 0
+        assertEq(join.bar(), 0);
+
+        // this should unwind the position
+        assertTrue(adai.balanceOf(addr.addr("MCD_JOIN_DIRECT_AAVEV2_DAI")) > 0);
+        join.exec();
+        assertEq(adai.balanceOf(addr.addr("MCD_JOIN_DIRECT_AAVEV2_DAI")), 0);
+
+        hevm.warp(DssSpell(spell).nextCastTime());
+        DssSpell(spell).cast();
         assertTrue(spell.done());
+    }
```

Comment:
Here is the test.  I also called the `exec()` in the test to ensure it unwinds the position.  There may be a more robust way to check the position (like checking the Vault `art`), but this is probably good enough.